index.html

```diff
@@ -0,0 +1,92 @@
+<!DOCTYPE html>
+<html lang="en">
+
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>For the Love of Coffee</title>
+    <link href="styles/style.css" rel="stylesheet">
+    <link href="styles/index.css" rel="stylesheet">
+    <link rel="icon" type="image/x-icon" href="images/logo.gif">
+  </head>
+  
+  <body>
+    
+    <header>
+        <nav>
+          <a href="index.html" class="active">Home</a>
+          <a href="pages/facts.html">Fun Facts</a>
+          <a href="pages/gallery.html">Gallery</a>
+          <a href="pages/about_us.html">About Us</a>
+        </nav>
+    </header>
+    
+    <main>
+      <h1> &#9749; For the Love of Coffee &#9749; 
+      </h1>
+      <section>
+          <img id="logo" src="./images/logo.gif" alt="logo">
+        
+          <h2> A dedicated place for coffee appreciation and its cultural significance.</h2>
+      </section>
+      
+      <div class="grid-container">
+        <section id="description1">
+          <h3> What <em>is</em> coffee? </h3>
+            <p>
+              Ahhhh, coffee. Coffee is an unofficial sponsor of weekdays.
+              It can turn the groggiest person into an alert and active one!
+            </p>
+            <p>
+              Okay, in all seriousness, coffee in our everyday sense is a beverage brewed from roasted coffee beans, and is prepared in several different ways
+              all across the world. Some people prefer it plain, while others prefer to add milk, cream, sugar, spices, and several other additions.
+              However, the National Coffee Association defines coffee as the berries harvested from a tropical plant.
+            </p>
+            <p>
+              Visit our Fun Facts page to learn more neat details about cofee. You'll also
+              find a neat collection of our favorite coffeehouses in our Gallery page!
+              To learn more about the people behind this fansite, dive into our About Us page.
+            </p>
```

Comment:
These pieces of text could also go in the same `p` tag since there aren't any different HTML elements found between them or any specified classes.